infra/terraform/external_function/snowalert/lambda-code/lambda_function.py

```diff
@@ -70,24 +69,4 @@ def lambda_handler(event, context=None):
             ]
         )
 
-    data_dumps = dumps({'data': res_data})
```

Comment:
looks like you're undoing some of the error handling I added. please keep it as-is.